backend/src/app/rpc/mutations/teams.clj

```diff
@@ -418,3 +430,24 @@
                 :email email
                 :role role)))
       team)))
+
+
+;; --- Mutation: Update invitation role
+
+(s/def ::update-team-invitation-role
+  (s/keys :req-un [::profile-id ::team-id ::email ::role]))
+
+(sv/defmethod ::update-team-invitation-role
+  [{:keys [pool] :as cfg} {:keys [profile-id team-id email role] :as params}]
+  (db/with-atomic [conn pool]
+    (let [perms    (teams/get-permissions conn profile-id team-id)
+          team     (db/get-by-id conn :team team-id)]
+
+      (when-not (:is-admin perms)
+        (ex/raise :type :validation
+                  :code :insufficient-permissions))
+
+      (db/update! conn :team_invitation
+                  {:role (name role) :updated_at (dt/now)}
+                  {:team_id (:id team) :email_to (str/lower email)})
```

Comment:
The same here: use `-` instead of `_`, they will be translated to `_` automatically for the database

Reply:
fixed